app/extras/model_locator.rb

```diff
@@ -5,6 +5,8 @@ def locate
       resource_class.find id_param
     elsif key_param = params[:"#{model}_key"]
       resource_class.find_by_key key_param
+    elsif model == :user
```

Comment:
I think it turns out we need to be able to search by username in order to make @mentions work, because the markdown renderer needs to come up with a url (or some user information), and has only the username to go off of.

Other option would be loading all the mentioned users with a comment and doing Records.find(username: (text)) somewhere, but I think that's too much for a mention-heavy thread.